bitrise.yml

```diff
@@ -0,0 +1,48 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  check:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git: { }
+
+  e2e:
+    steps:
+    - git::https://github.com/bitrise-steplib/steps-check.git:
+        inputs:
+        - workflow: e2e
+
+  sample:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - TEST_APP_BRANCH: master
+    - BITRISE_PROJECT_PATH: ./_tmp/BullsEye.xcworkspace
+    - BITRISE_SCHEME: BullsEye
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -xeo pipefail
+            rm -rf "./_tmp"
+    - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git:
+        inputs:
+        - repository_url: $TEST_APP_URL
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - git::https://github.com/bitrise-steplib/steps-xcode-build-for-test.git@test_plan_output_export:
```

Comment:
Why does the sample workflow use the xcode build for test step from a specific branch while in the e2e test you are taking the the step from the steplib?

Reply:
Forgot to update the `sample` workflow, fixed in https://github.com/bitrise-steplib/bitrise-step-xcode-test-without-building/pull/1/commits/d7b3b1f4b92e4a2a9963c194c5a53b2a9fa7de40